src/Test.Utilities/VisualBasicCodeFixVerifier`2.cs

```diff
@@ -23,14 +24,18 @@ public static DiagnosticResult Diagnostic(string diagnosticId)
         public static DiagnosticResult Diagnostic(DiagnosticDescriptor descriptor)
             => VisualBasicCodeFixVerifier<TAnalyzer, TCodeFix, XUnitVerifier>.Diagnostic(descriptor);
 
-        public static async Task VerifyAnalyzerAsync(string source, params DiagnosticResult[] expected)
+        public static Task VerifyAnalyzerAsync(string source, params DiagnosticResult[] expected)
+            => VerifyAnalyzerAsync(source, default(Action<Test>), expected);
+
+        public static async Task VerifyAnalyzerAsync(string source, Action<Test> setup, params DiagnosticResult[] expected)
```

Comment:
❕ Do not add this overload. It is already directly supported by the test framework:

```csharp
await new VerifyVB.Test
{
  TestCode = code,
  TestBehaviors = TestBehaviors.SkipGeneratedCodeCheck,
}.RunAsync();
```